modules/canvaskit/canvaskit_bindings.cpp

```diff
@@ -1674,6 +1674,17 @@ EMSCRIPTEN_BINDINGS(Skia) {
         .function("_flush", optional_override([](SkSurface& self) {
             self.flushAndSubmit(false);
         }))
+        .function("_getTextureId", optional_override([](SkSurface& self) -> GrGLuint {
+            GrBackendTexture texture = self.getBackendTexture(SkSurface::kFlushRead_BackendHandleAccess);
+            if (!texture.isValid()) {
+                return 0;
```

Comment:
Just to be sure. Is it 0 in case of failure or null or -1 or undefined?

Reply:
I can't return null or undefined or -1, since I'm returning a GrGLuint?
I think 0 is a good return value for an invalid texture id, since it's equivalent to not having a texture bound.

Reply:
ok Just checking for type correctness